tools/miqldap_to_sssd/cli.rb

```diff
@@ -0,0 +1,57 @@
+require 'trollop'
+
+module MiqLdapToSssd
+  class Cli
+    attr_accessor :options
+
+    def parse(args)
+      args.shift if args.first == "--" # Handle when called through script/runner
+
+      LOGGER.debug("Invokded #{self.class}\##{__method__}")
+
+      self.options = Trollop.options(args) do
+        banner "Usage: ruby #{$PROGRAM_NAME} [options] database [...]\n"
+
+        opt :dry_run, "Dry Run", :short => "d"
+
+        opt :basedn_domain,
+            "The Base DN domain name, e.g. example.com",
+            :short   => "D",
+            :default => nil,
+            :type    => :string
+
+        opt :tls_cacert,
+            "Path to certificate file",
+            :short   => "c",
+            :default => nil,
+            :type    => :string
+
+        opt :only_change_userids,
+            "change the userids then exit",
+            :short   => "n",
+            :default => false,
+            :type    => :flag
+
+        opt :skip_post_coversion_userid_change,
+            "Do the MiqLdap to SSSD conversion but skip the normalizing of the userids",
+            :short   => "s",
+            :default => false,
+            :type    => :flag
```

Comment:
In a followup, let's create a separate executable in the bin directory that will invoke this script and allow a user to just run "only_change_userids"

Reply:
@jrafanie Thank you! I've updated PR https://github.com/ManageIQ/manageiq-appliance/pull/134 to include the requested executable script.